config/routes.rb

```diff
@@ -11,11 +11,12 @@
     end
   end
 
-  get 'sign_in' , to: 'sessions#new'
-  get 'sign_out', to: 'sessions#destroy'
-  get 'home',     to: 'pages#home'
-  get 'register', to: 'users#new'
-  get 'my_queue', to: 'queue_items#index'
+  get  'sign_in' , to: 'sessions#new'
+  get  'sign_out', to: 'sessions#destroy'
```

Comment:
Solution says `get` but i would recommend `delete` as thats more RESTful approach also, users should not be given a chance to bookmark destructive actions.